### ISSUE TITLE
test for same value

### ISSUE THREAD

**Issue by zhoususheng** [dmlc/xgboost#1588]
Dear chen:
After I train a xgboost model, I test on my test set, but I get same value for the different test record. I do this on sklearn gradient boostring regressor, I get the result. So I want ask you, why I get the same value using xgboost, and how should I tune parameters?
Thank you very much!

**Comment by pommedeterresautee** [comment]
Old question, no answer. Closing
